lib/inspec/reporters/base.rb

```diff
@@ -5,9 +5,36 @@ class Base
     def initialize(config)
       @config = config
       @run_data = config[:run_data]
+      apply_options_to_run_data
```

Comment:
I know naming things is hard and no one will ever agree, but perhaps "apply_message_size_limits"? If there are future options to implement, they would go in their own methods.

Reply:
I was foreseeing the opposite - that this method _would_ handle any future options. I can rename the method like you suggest so that it is named specific to the thing it _currently_ does, and we can discuss handling future options when we get to that point.